{
    "ErrorBoundary.errorOccured": "एक अप्रत्याशित त्रुटि हुई।"
}